content/en/serverless/_index.md

```diff
@@ -29,27 +29,36 @@ Serverless is a concept where you write event-driven code and upload it to a clo
 
 ### 1. Install the AWS Integration
 
-Start by installing the [AWS Integration][2]. This allows Datadog to ingest Amazon CloudWatch metrics from AWS Lambda. The AWS integration installation also configures the Datadog Forwarder, which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
+Install the [AWS integration][2]. This allows Datadog to ingest Lambda metrics from AWS CloudWatch. 
 
-### 2. Instrument your application
+### 2. Install the Datadog Forwarder
+
+Install the [Datadog Forwarder Lambda function][3], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
+
+Note, if you installed the AWS integration by using the Datadog-provided CloudFormation template, then you already have the Forwarder function installed as part of the AWS integration CloudFormation stack.
```

Comment:
Can we modify the wording here so this step isn't followed by customers who installed the AWS integration? Maybe as part of section 1, we could have "If you did installed the AWS integration using the Datadog-provided CloudFormation template, install the [Datadog Forwarder Lambda function][3], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs."